packages/botonic-react/src/webchat-app.tsx

```diff
@@ -360,10 +496,15 @@ export class WebchatApp {
   }
 
   isOnline() {
-    return this.webchatRef.current.isOnline()
+    return this.webchatRef.current?.isOnline()
   }
 
-  async resolveWebchatVisibility(optionsAtRuntime) {
+  async resolveWebchatVisibility(optionsAtRuntime?: {
+    appId: string
+    visibility: boolean | (() => void) | 'dynamic' | undefined
+  }) {
+    if (!optionsAtRuntime) return true // If optionsAtRuntime is not provided, always render the webchat
```

Comment:
review what happen if `{}` is passed

Reply:
But you could have the appID defined in the src/webchat/index.ts file of the bot and this would be used in the WebchatApp constructor